save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/OrganizationRepository.kt

```diff
@@ -30,4 +33,18 @@ JpaSpecificationExecutor<Organization> {
      * @return list of organization by owner id
      */
     fun findByOwnerId(ownerId: Long): List<Organization>
+
+    /**
+     * @param organizationName
+     * @return 1 if [organizationName] is valid, 0 otherwise
+     */
+    @Query("""select if (count(*) = 0, true, false) from save_cloud.high_level_names where name = :org_name""", nativeQuery = true)
+    fun validateOrganizationName(@Param("org_name") organizationName: String): Long
```

Comment:
maybe add `HighLevelNamesRepository` and use methods from them? also, why it returns Long? Doesn't Spring support converting for boolean?

Reply:
I guess we do not need repository for that because as far as I understrand, repository means creating entity.

And fun thing is that Long is not converted to boolean. Tried it and it failed.

Reply:
`HighLevelNamesRepository` is simplification: we can create an entity with a single field or write such annotation with query on each repository which has to have unique name.
but it's up to you